package/lite/etc/addons/cisco/app-netsource-cisco_esa.conf

```diff
@@ -38,7 +38,8 @@ filter f_cisco_esa_authentication {
     or match('failed\s*authentication.', value("MESSAGE"))
     or match('Time\s*offset\s*from\s*UTC:', value("MESSAGE"))
     or match('[vV]ersion:\s+.*\s+SN:\s+.*', value("MESSAGE"))
-    or match('\s*login|logout:\s*[^\s]*\s*[Uu]ser:[\w\-]+\s*session:[^\s]+', value("MESSAGE"))
+    or match('\s*(?:login|logout):\s*[^\s]*\s*[Uu]ser:[\w\-]+\s*session:[^\s]+', value("MESSAGE"))
```

Comment:
is there a reason we are making it non capturing ?

Reply:
without capturing group previous regex matches only login keyword instead of matching the whole sequence of regex.